### DISCUSSION TITLE
create test account

### DISCUSSION THREAD

**Discussion by Vaibhavipatel15** [amzn/ads-advanced-tools-docs#112] [Q&A]
How to create test account in amazon ads API?

**Comment by jerirose** [comment]
Hi @Vaibhavipatel15 , you can find an overview of Amazon Ads API test accounts in the advanced tools center: https://advertising.amazon.com/API/docs/en-us/guides/account-management/test-accounts/overview

If you have been approved for API access, you can use your client application to create test accounts by following the setup guide in the advanced tools center, starting with [Authorization for test accounts](https://advertising.amazon.com/API/docs/en-us/guides/account-management/test-accounts/authorization-for-test-accounts).

Requests for the `/testAccounts` endpoint are also included in the [Amazon Ads API Postman collection](https://github.com/amzn/ads-advanced-tools-docs/tree/main/postman) in this repository. The **Test accounts** folder in the collection includes inline documentation for setting up your Postman environment to use test accounts.

Note that an additional permission scope, `advertising::test:create_account`, is required to create test accounts, and that the authorization grant should be provided by an Amazon user account that does _not_ currently manage Amazon Ads accounts.
